package.json

```diff
@@ -200,10 +200,5 @@
     "@axelar-network/axelarjs-types@^0.33.0": "patch:@axelar-network/axelarjs-types@npm%3A0.33.0#./.yarn/patches/@axelar-network-axelarjs-types-npm-0.33.0-5d36465d11.patch"
   },
   "private": true,
-  "packageManager": "yarn@3.5.0",
-  "expo": {
```

Comment:
There are two ways to link the native modules on Expo:
-  add native modules in ./modules folder then expo is autolink it by default.
- another adding autolinking config on package.json

Previously I used ./modules folder for weshd; it had some issues on the linking on github ci so I have used package.json method.

Doing this; it will lead to error due to below code
`const WeshnetModule = require("../../weshd")` in packages/weshnet/services.ts